django-stubs/db/backends/mysql/base.pyi

```diff
@@ -39,20 +40,20 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     validation_class: type[DatabaseValidation]
 
     vendor: str
-    data_types: Any
-    operators: Any
+    data_types: dict[str, str]
+    operators: dict[str, str]
     pattern_esc: str
-    pattern_ops: Any
-    isolation_levels: Any
+    pattern_ops: dict[str, str]
+    isolation_levels: set[str]
     Database: Any
-    SchemaEditorClass: Any
-    isolation_level: Any
+    SchemaEditorClass: type[DatabaseSchemaEditor]
+    isolation_level: set[str]
     def get_connection_params(self) -> dict[str, Any]: ...
     def get_new_connection(self, conn_params: Any) -> Any: ...
     def init_connection_state(self) -> None: ...
     def create_cursor(self, name: Any | None = ...) -> CursorWrapper: ...
     def disable_constraint_checking(self) -> Literal[True]: ...
-    needs_rollback: Any
+    needs_rollback: bool
```

Comment:
Where possible, please just remove attributes that are present in the base class and have the same definition.

`BaseDatabaseWrapper` already defines `needs_rollback: bool`, so this can be removed here.